sequence.go

```diff
@@ -89,6 +89,19 @@ func PyByteArray_AsString(self *PyObject) string {
 	return C.GoString(c_str)
 }
 
+// Return the contents of bytearray as []bytes
+func PyByteArray_AsBytes(self *PyObject) []byte {
+	length := C._gopy_PyByteArray_GET_SIZE(topy(self))
+	c_str := C.PyByteArray_AsString(topy(self))
+	return C.GoBytes(unsafe.Pointer(c_str),C.int(length))
+}
+
+// Return the contents of bytearray as []bytes, size length
+func PyByteArray_AsBytesN(self *PyObject, length int) []byte {
+	c_str := C.PyByteArray_AsString(topy(self))
+	return C.GoBytes(unsafe.Pointer(c_str),C.int(length))
```

Comment:
perhaps check that the passed `length` is within the allowed range?
ie: call `_gopy_PyByteArray_GET_SIZE` and compare (+panic with out of range)